python/pyspark/testing/utils.py

```diff
@@ -209,3 +234,70 @@ def check_error(
         self.assertEqual(
             expected, actual, f"Expected message parameters was '{expected}', got '{actual}'"
         )
+
+
+def assertSchemaEqual(
```

Comment:
I don't think you should put it in a separate method. Just put inside to `assertSchemaEqual` directly

Reply:
@HyukjinKwon should we also expose the assertSchemaEqual function, if users want it?

Reply:
We could do but not sure if users want this since it's fairly simple. I think this would even work with unittests' assertEqual (?)

Reply:
yep sounds good